spk/vim/Makefile

```diff
@@ -1,9 +1,12 @@
 SPK_NAME = vim
 SPK_VERS = 9.0
-SPK_REV = 5
+SPK_REV = 6
 SPK_ICON = src/vim.png
 
-DEPENDS = cross/vim
+DEPENDS = cross/$(SPK_NAME)
+PYTHON_PACKAGE = python311
+SPK_DEPENDS = "python311>=3.11.5-8"
+UNSUPPORTED_ARCHS = $(OLD_PPC_ARCHS) $(ARMv5_ARCHS)
 
 MAINTAINER = SynoCommunity
```

Comment:
Since python is an optional dependency for the vim package, we must not add "SPK_DEPENDS"

Reply:
@th0ma7 we could either use python310 for those archs or drop python support. 

We should keep off the hardcoded dependency of python, since python support is optionally and must be manually configured by the user.

Reply:
@th0ma7 probably I am wrong with the "manually configure vim for python" but vim can be built with or without the python feature.

Reply:
@hgy59 do you mind if we revisit this to a later time?  I agree that it can certainly be worked out differently, and would benefit from an update also.  But for the sake of py311 upgrade I'd keep it as is...?

Reply:
@th0ma7 if you want to postpone this, then please leave PYTHON_PACKAGE for vim at python310 to keep the packages for OLD_PPC_ARCHS and ARMv5_ARCHS.

And please remove SPK_DEPENDS.